main.js

```diff
@@ -15,6 +15,14 @@ const popUp = document.querySelector(".desk-popup");
 const closeButton2 = document.querySelector(".close-popup");
 const professionalArtDiv = document.querySelector(".professional-art");
 
+const error = document.getElementById("error");
+const submitForm = document.getElementById("form");
+const submitBtn = document.getElementById("btn__submit");
+
+// form data
+const email = document.getElementById("email");
+const fullName = document.getElementById("full-name");
+const textArea = document.getElementById("textarea");
```

Comment:
- [ ] As required for this project The form data for all inputs should be stored in a single object, and saved to a single local storage registry. have a look below how it should be :point_down: 
```

const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

Reply:
who is @Rikol137??